pkg/app/mocks_test.go

```diff
@@ -113,3 +119,7 @@ func (helm *noCallHelmExec) IsVersionAtLeast(versionStr string) bool {
 	helm.doPanic()
 	return false
 }
+
+func (helm *noCallHelmExec) ShowChart(chartPath string) (chart.Metadata, error) {
+	return chart.Metadata{}, errors.New("fake test error")
```

Comment:
helm.doPanic()
return chart.Metadata{}, nil

Reply:
and this @felipecrs

Reply:
`doPanic()` here is fine

Reply:
changed, thanks